examples/tutorials/hello-gltf/index.html

```diff
@@ -21,14 +21,26 @@
       transition: opacity 1s ease-in-out;
     }
 
-    #model-select {
+    #model-options {
```

Comment:
Nit: I feel like either we create UI components in the code or in the HTML. The mixed approach doesn't simplify things IMHO. I suggest moving to programmatic creation.

Since you already building. a function to create select elements then just call

```ts
createSelect({root=..., id='model-options', label='Animation', defaultValue='camera', options={
  camera: 'Camera-Animation', gltf: 'glTF-Animation', pbr: 'PBR', onChange=()=>{}});
```

Reply:
ok, let me do it in a future PR, when I need more ui elements.